.env.development

```
@@ -0,0 +1,48 @@
+# Development Environment Configuration for FinTrac
+# This file contains environment variables for local development
+
+# Database Configuration - Use PouchDB with CouchDB sync
+VITE_USE_POUCHDB=false
+
+# CouchDB Configuration (for network access)
+VITE_SYNC_ENABLED=true
+VITE_COUCHDB_URL=http://10.196.88.186:5984
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_COUCHDB_URL key should go before the VITE_SYNC_ENABLED key

(UnorderedKey)